draft-rokui-ccamp-actn-wdm-pluggable-modelling.md

```diff
@@ -461,84 +448,55 @@ Coherent pluggable have revolutionized optical networking by offering a powerful
-* operational-mode: This is the main part of this document which contains a list of capability attributes supported by the coherent pluggable.
+{: #plug-config-attribute}
+## Coherent Pluggable Configurations Attributes
 
-{{plug-manifest}} discusses the concept of the "coherent pluggable manifest", which is a repository for all supported-modes". It also outlines the benefit of such repository and various use-cases.
-Some details of supported modes can be found in IETF draft {{Impairment}}.
+Referring to {{figure-plug-config}}, the coherent pluggables support a set of read-write attributes which are configurable. Example of such configuration attributes are output power, central frequency and operational-mode. Note that as discussed in {{plug-capabilities-attributes}}, since a coherent pluggable may support multiple operational-modes, as part of these configuration attributes, operator should configure which of these operational-mode is desired and should be functional.
 
 ~~~~
 
  |-----------------------------------------------------------------|
- |  supported-mode*    // list of supported-modes                  |
- |      supported-mode-id                                          |
- |      organization-id (e.g., ITU-T, OIF, OpenConfig or Vendor)   |
- |      tag* // Optional. It has various usage in future           |
- |      operational-mode                                           |
- |              capability-attribute-1                             |
- |              capability-attribute-2                             |
- |              ....                                               |
- |              capability-attribute-n                             |
+ |  optical-channel  // OTSi channels                              |
+ |      configuration // list of R/W plug configuration attributes |
+ |           config-attribute-1                                    |
+ |           config-attribute-2                                    |
+ |           .....                                                 |
+ |           config-attribute-m                                    |
  |-----------------------------------------------------------------|
 
 ~~~~
-{: #figure-plug-supported-mode title="Data structure for Coherent pluggable supported-modes"}
-
-As an example, the operational-mode 0x3E introduced by OIF contains the following read-only capability attributes. Notes that in general a coherent pluggable can support multiple operational modes.
-
-~~~~
-organization-id:  OIF
-operational-mode: 0x3E
-      modulation: DP-16QAM
-      bit-rate: 478.75 Gbps
-      baud-rate: 59.84375 Gbd
-      more attributes ...
-~~~~
+{: #figure-plug-config title="Data structure for Coherent pluggable Configuration Attributes"}
```

Comment:
{: #figure-plug-config title="Data structure for optical module Configuration Attributes"}